lib/streamlit/watcher/file_watcher.py

```diff
@@ -50,6 +39,24 @@
 ]
 
 
+def report_watchdog_availability():
+    if watchdog_available == False:
```

Comment:
nit: I think writing

```python
if not watchdog_available:
    # ...
```
is a bit more pythonic

Reply:
updated